tools/c2chapel/c2chapel.py

```diff
@@ -84,7 +84,7 @@
 c2chapel["uintptr_t"]          = "c_uintptr"
 c2chapel["ptrdiff_t"]          = "c_ptrdiff"
 c2chapel["ssize_t"]            = "ssize_t"
```

Comment:
Sorry I didn't catch this before seeing the merge message:  `ssize_t` should similarly be updated to `c_ssize_t`.

Reply:
Wow, I was looking right at that PR that made these changes and still misses this... should be fixed by https://github.com/chapel-lang/chapel/pull/20836